templates/common/_base/files/etc-systemd-system.conf.d-10-default-env-proxy.conf.yaml

```diff
@@ -0,0 +1,9 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/systemd/system.conf.d/10-default-env-proxy.conf"
+contents:
+  inline: |
+    {{if .Proxy -}}
+    [Manager]
+    DefaultEnvironment={{if .Proxy.HTTPProxy}}HTTP_PROXY={{.Proxy.HTTPProxy}} {{end}}{{if .Proxy.HTTPSProxy}}HTTPS_PROXY={{.Proxy.HTTPSProxy}} {{end}}{{if .Proxy.NoProxy}}NO_PROXY={{.Proxy.NoProxy}}{{end}}
```

Comment:
Rather than wrap each in the if, it might make more sense to just put that immediately after the if .Proxy once.

Reply:
Like this?

```suggestion
    {{if .Proxy -}}
    [Manager]
    DefaultEnvironment=HTTP_PROXY={{.Proxy.HTTPProxy}} HTTPS_PROXY={{.Proxy.HTTPSProxy}} NO_PROXY={{.Proxy.NoProxy}}
```